includes/Story_Revisions.php

```diff
@@ -251,17 +251,17 @@ public function enqueue_player_script(): void {
 
 		wp_add_inline_script(
 			AMP_Story_Player_Assets::SCRIPT_HANDLE,
-			<<<JS
-	const loadPlayers = () => document.querySelectorAll('amp-story-player').forEach(playerEl => (new AmpStoryPlayer(window, playerEl)).load());
-
-	const originalFrame = wp.revisions.view.Frame;
-	wp.revisions.view.Frame = originalFrame.extend({
-		render: function() {
-			originalFrame.prototype.render.apply(this, arguments);
-			loadPlayers();
-			this.listenTo( this.model, 'update:diff', () => loadPlayers() );
-		},
-	});
+			<<<'JS'
+    const loadPlayers = () => document.querySelectorAll('amp-story-player').forEach(playerEl => (new AmpStoryPlayer(window, playerEl)).load());
+
+    const originalFrame = wp.revisions.view.Frame;
+    wp.revisions.view.Frame = originalFrame.extend({
+        render: function() {
+            originalFrame.prototype.render.apply(this, arguments);
+            loadPlayers();
+            this.listenTo( this.model, 'update:diff', () => loadPlayers() );
+        },
+    });
```

Comment:
Is this change in indentation intentional?

Reply:
I believe it is this lint. https://github.com/slevomat/coding-standard/blob/master/doc/php.md#slevomatcodingstandardphprequirenowdoc-

Reply:
Does the linter complain if you undo the indentation change?

Reply:
the linter indented with when I run lint:php:fix. What is wrong with the indentation?

Reply:
I just want to better understand why this happens and whether this type of indentation (spaces vs. tabs) is something that's enforced by the fixer, as this type of mixed indentation is a bit confusing and unexpected.

Reply:
Just tested this and I don't get any indentation changes with the current PHPCS config, so I would suggest to undo this unrelated change that is not enforced by the fixer.